cmd/podman/main_local.go

```diff
@@ -31,7 +32,37 @@ import (
 
 const remote = false
 
+func runRemote(path string, args ...string) (int, error) {
+	c := exec.Command(path, args...)
+	c.Stdin = os.Stdin
+	c.Stdout = os.Stdout
+	c.Stderr = os.Stderr
+	if err := c.Run(); err != nil {
+		if exitError, ok := err.(*exec.ExitError); ok {
+			waitStatus := exitError.Sys().(syscall.WaitStatus)
+			return waitStatus.ExitStatus(), nil
+		}
+		return 1, err
+	}
+	return 0, nil
+}
+
 func init() {
+	host := os.Getenv("PODMAN_HOST")
+	bridge := os.Getenv("PODMAN_VARLINK_BRIDGE")
+	address := os.Getenv("PODMAN_VARLINK_ADDRESS")
+	if host != "" || bridge != "" || address != "" {
+		program := "podman-remote"
```

Comment:
It might be good to have a debug statement in here to just say "Attempting to connect to remote Podman" or some such.  I could see it as being potentially helpful in a debugging situation later on.

Reply:
So something like a `logrus.Debugf` with the command being executed perhaps.

Reply:
Tried adding logging, but it doesn't show up in the logs (at the debug level)

Reply:
@TomSweeneyRedHat : apparently this (`init`) happens _before_ logging has been set up

That is, we are still running with the default log level of InfoLevel (hiding the debug logging)